source/retryable-reads/tests/distinct.yml

```
@@ -0,0 +1,71 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "Distinct succeeds on first attempt"
+        clientOptions:
+            retryReads: true
+        operation:  &retryable_operation
+            name: distinct
+            object: collection
+            arguments: { fieldName: "x", filter: { _id: { $gt: 1 } } }
+            result:
+                - 22
+                - 33          
+        expectations:
+            - &retryable_command_started_event
+                command_started_event:
+                    command:
+                        distinct: *collection_name
+                        key: "x"
+                        query: 
+                            _id: {$gt: 1}
+                    database_name: *database_name
+    -
+        description: "Distinct succeeds on second attempt"
+        clientOptions:
+            retryReads: true
+        failPoint:  &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands: [distinct]
+                closeConnection: true
+        operation: *retryable_operation                
+        expectations:
+             - *retryable_command_started_event
```

Missing the second command:
```suggestion
             - *retryable_command_started_event
             - *retryable_command_started_event
```